test/new_relic/agent/serverless_handler_test.rb

```diff
@@ -46,6 +87,8 @@ def teardown
       # integration style
 
       def test_the_complete_handoff_from_the_nr_lambda_layer
+        skip 'This serverless test is limited to Ruby v3.2+' unless ruby_version_float >= 3.2
```

Comment:
Should the whole file be skipped unless Ruby > 3.2? Would this work in setup?

Reply:
That's a good idea. All tests now skipped via [839bb1b](https://github.com/newrelic/newrelic-ruby-agent/pull/2811/commits/839bb1b0e777c4dbe5d710b99efce296de5cc4d3)